# リモートモードのON/OFF
bool remote_control_mode
